frontend/cypress/support/commands.ts

```diff
@@ -183,6 +183,15 @@ Cypress.Commands.add('login', (username: string, password: string) => {
           .then(() => {
             haveCookie = true;
           });
+      } else if (auth_strategy === 'token') {
+        cy.exec('kubectl exec deploy/kiali -n istio-system -- cat /var/run/secrets/kubernetes.io/serviceaccount/token')
+          .then(result => {
+            cy.visit('/');
+            cy.get('#token').type(result.stdout, { log: false });
+            cy.get('button[type=submit]').click();
+            cy.get('#page-sidebar').should('be.visible');
+            haveCookie = true;
```

Comment:
If you login programmatically through the API using `cy.request` like in the openshift login above, you should end up with a valid kiali token cookie. iirc `cy.request` will automatically save the cookie when it gets a `Set Cookie` header. The steps should be MUCH simpler than the openshift steps though for token auth. Hopefully just 1 request to get the cookie given a valid token.

Reply:
ooh, awesome info Nick, Edgar is on PTO today but should be able to proceed on Monday.

Reply:
OK, I did auth via the API and now it looks stable.